packages/slack-app/src/client/index.ts

```diff
@@ -117,7 +117,7 @@ export default function createAlvtimeClient(
 
   return {
     getTasks(accessToken: string) {
-      return fetcher(concatURL("/api/user/tasks"), {
+      return fetcher(concatURL("/api/user/Tasks"), {
```

Comment:
Unsure why this is suddenly camelCased, looks weird for an api. I would say its better to keep api url's all lowercase.